src/bindings/gsettings/CMakeLists.txt

```diff
@@ -48,7 +48,7 @@ if (CMAKE_VERSION GREATER 3.3)
 	pkg_get_variable (GIO_MODULE_DIR gio-2.0 giomoduledir)
 else ()
 	execute_process (
-		COMMAND PKG_CONFIG_EXECUTABLE --variable=giomoduledir
+		COMMAND PKG_CONFIG_EXECUTABLE --variable=giomoduledir gio-2.0
```

Comment:
I think this alone wont fix the issue, execute_process is not used correctly, see https://cmake.org/cmake/help/v3.0/command/execute_process.html.